### PR TITLE
Update SDK and test for BatchAI for GA

### PR DESCRIPTION
**Pull request by llidev**
## Description
Onboarding of .NET SDK for Azure BatchAI Service and related tests for GA. This is being generated by the following spec: https://github.com/Azure/azure-rest-api-specs/tree/master/specification/batchai/resource-manager/Microsoft.BatchAI/stable/2018-03-01

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks like you are making some breaking changes here.
Please update the package version number in csproj file
(`1.1.0-preview`)
Also, please create a `generate.ps1` file similar to [this](https://github.com/lliimsft/azure-sdk-for-net/blob/8fb11dd887fe2588fc47cefc59a818d1e91eef60/src/SDKs/Compute/Management.Compute/generate.ps1) and generate the code again using this script

**Comment by xingwu1** [comment]
@dsgouda I don't think we made breaking change. We only add 2 new APIs (if that counts breaking change).

File: src/SDKs/BatchAI/Management.BatchAI/Generated/Models/JobsListOutputFilesOptions.cs [hunk 1]
```diff
@@ -55,21 +57,27 @@ public JobsListOutputFilesOptions(string outputdirectoryid, int? linkexpiryinmin
         /// OutputDirectory--&amp;gt;id parameter that is given by the user
         /// during Create Job.
         /// </summary>
-        [JsonProperty(PropertyName = "")]
+        [Newtonsoft.Json.JsonIgnore]
```
Comment (dsgouda) [inline src/SDKs/BatchAI/Management.BatchAI/Generated/Models/JobsListOutputFilesOptions.cs]: @shahabhijeet Just want to confirm if this is a breaking change

**Review by dsgouda — CHANGES_REQUESTED** [review]

**Comment by dsgouda** [comment]
We need to confirm whether the changes to `JsonProperty` is something introduced in the AutoRest generator. Will approve and merge once done.

File: src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj [hunk 2]
```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.BatchAI</PackageId>
 		<Description>Provides management capabilities for Azure Batch AI.</Description>
-		<Version>1.0.0-preview</Version>
```
Comment (dsgouda) [inline src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj]: Is this meant to be a GA package? If yes, please update the `PackageReleaseNotes`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Confirmed that the generated code is as expected. Thanks for addressing the comments.
Looks good apart from the minor comment

File: src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj (hunk 2 above)
Comment (lliimsft) [reply src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj]: PackageReleaseNotes have been updated

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**